src/schnorr.rs

```diff
@@ -7,13 +7,12 @@ use rand::thread_rng;
 #[cfg(any(test, feature = "rand"))]
 use rand::{CryptoRng, Rng};
 
-use super::Error::{InvalidPublicKey, InvalidSecretKey, InvalidSignature};
+use super::Error::InvalidSignature;
```

Comment:
Just FYI, but there was a similar discussion at the time of adding the schnorr module and it was decided to use the same name as upstream at the end: https://github.com/rust-bitcoin/rust-secp256k1/pull/237#issuecomment-715452744
It was also decided to use `schnorrsig` instead of `schnorr`: https://github.com/rust-bitcoin/rust-secp256k1/pull/237#issuecomment-716457800
(I don't personally have a strong opinion)